scripts/home.js

```diff
@@ -125,6 +125,16 @@ async function setupHomepage() {
     }
   });
 
+  // add publish dependencies
+  window.hlx=window.hlx||{};
+  window.hlx.dependencies = [];
+  const limit = 256;
+  let offset = 0;
+  while (offset < limit * 10) {
```

Comment:
This will only do the 10 first, no ?
Maybe the 10 should be a global variable.

Reply:
Yes, I didn't want to purge too many... 

Where would you put it as a global variable?

Reply:
somewhere in the file, just to make the "10" or more meaningful...

Reply:
If you do not purge all, some articles disappear from the list.

Reply:
As long as you add to the list, the only risk woyuld be duplicate cards in the first unpurged segment, but I think we have a check for that before we add a new card.